openfisca_us/variables/gov/states/md/tax/income/exemption/md_aged_blind_exemptions.py

```diff
@@ -0,0 +1,58 @@
+from openfisca_us.model_api import *
+
+
+class md_aged_blind_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "MD aged blind exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://govt.westlaw.com/mdc/Document/NF59A76006EA511E8ABBEE50DE853DFF4?viewType=FullText&originationContext=documenttoc&transitionType=CategoryPageItem&contextData=(sc.Default)"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        filing_status_type = filing_status.possible_values
+
+        blind = parameters(period).gov.states.md.tax.income.exemptions.blind
+        # Count number of is_blind from tax_unit
+        blind_head = tax_unit("blind_head", period) * 1
```

Comment:
Just ran a [speed test](https://colab.research.google.com/drive/1Sz73GZuBhNObnW700K6YQTg31O4YHQ4h?usp=sharing) showing this is faster
```suggestion
        blind_head = tax_unit("blind_head", period).astype(int)
```
```suggestion
        blind_head = tax_unit("blind_head", period) * 1
```